examples/electron/package-lock.json

```diff
@@ -1,18 +1,38 @@
 {
-	"name": "desktopJS",
-	"version": "0.0.0",
-	"lockfileVersion": 1,
-	"requires": true,
-	"dependencies": {
-		"@morgan-stanley/desktopjs": {
-			"version": "3.9.0",
-			"resolved": "https://registry.npmjs.org/@morgan-stanley/desktopjs/-/desktopjs-3.9.0.tgz",
-			"integrity": "sha512-bxW7SkOL7xcwU2yPSzyQbmijUEpCf0CS09CVJ0TShTy3GLPjjItk9VFBklOEvEBmRIxCWcNsXQl7Om9/a/m5WQ=="
-		},
-		"@morgan-stanley/desktopjs-electron": {
-			"version": "3.9.0",
-			"resolved": "https://registry.npmjs.org/@morgan-stanley/desktopjs-electron/-/desktopjs-electron-3.9.0.tgz",
-			"integrity": "sha512-CtXXBO+MTm/1JLrOlqi4YA/9i337fE8JlTJelA1Up1O4SrTcnsyb1MY9GcGT4ivdnOccSZn2zaNpEUoMJssacg=="
-		}
-	}
+  "name": "desktopJS",
+  "version": "0.0.0",
+  "lockfileVersion": 2,
```

Comment:
@Brookke  I think this is the npm 7 issue again on this one

Reply:
Thanks will fix that